.prettierrc.js

```diff
@@ -2,5 +2,12 @@ module.exports = {
   ...require("@stellar/prettier-config"),
   // This is mostly content, and prose wrap has a way of exploding markdown
   // diffs. Override the default for a better experience.
-  proseWrap: "never",
+  overrides: [
+    {
+      files: "*.mdx",
+      options: {
+        proseWrap: "never",
+      },
+    },
+  ],
```

Comment:
Dang I thought spreading the stellar config would have been fine. Thanks for digging in